nix-index-wrapper.nix

```diff
@@ -1,36 +1,30 @@
-{ runCommand
-, makeWrapper
+{ lib
+, symlinkJoin
+, makeBinaryWrapper
 , nix-index-unwrapped
 , nix-index-database
+,
 }:
-runCommand "nix-index-with-db-${nix-index-unwrapped.version}"
```

Comment:
The reason to use `symlinkJoin` over `runCommand` is `symlinkJoin` will automatically propagate all files while `runCommand` you have to manually do it